games-strategy/lgeneral/lgeneral-1.3.1.recipe

```diff
@@ -0,0 +1,86 @@
+SUMMARY="A turn-based strategy engine"
+DESCRIPTION="LGeneral is a turn-based strategy engine heavily inspired by Panzer General. \
+You play single scenarios or whole campaigns turn by turn against a human \
+player or the AI. Entrenchment, rugged defense, defensive fire, surprise \
+contacts, surrender, unit supply, weather influence, reinforcements and other \
+implementations contribute to the tactical and strategic depth of the game."
```

Comment:
There is a recommendation to try and keep lines lengths <= 80 characters, with the exception of **`SOURCE_URI*`** and **`CHECKSUM_SHA256*`**. I don't know, however, if this recommendation is still desirable for existing recipes that do not _already respect_ it, but I guess that it would be nice to follow that rule for new recipes.
So here's the DESCRIPTION variable, re-wrapped for 80-char lines, in case you would like to follow that recommendation ;-)
```
DESCRIPTION="LGeneral is a turn-based strategy engine heavily inspired by \
Panzer General. You play single scenarios or whole campaigns turn by turn \
against a human player or the AI. Entrenchment, rugged defense, defensive \
fire, surprise contacts, surrender, unit supply, weather influence, \
reinforcements and other implementations contribute to the tactical and \
strategic depth of the game."
```
Yep, I know that's an easy task that anyone can do. I'm just copy/pasting it here for your convenience ;-)

Hmm, for the COPYRIGHT, maybe **`2000-2015`** would be better than **`2000, 2015`**, unless, of course, the software did not evolve between 2001 and 2014 ;-)